hmda/src/main/scala/hmda/validation/engine/2022/LarEngine2022.scala

```diff
@@ -0,0 +1,302 @@
+package hmda.validation.engine
+
+// $COVERAGE-OFF$
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.context.ValidationContext
+import hmda.validation.rules.lar.quality._2019._
+import hmda.validation.rules.lar.quality.{common, _2020 => quality2020, _2021 => quality2021, _2022 => quality2022}
+import hmda.validation.rules.lar.quality.common._
+import hmda.validation.rules.lar.syntactical.{S300, S301}
+import hmda.validation.rules.lar.validity.{_2020, _2022, _}
+import hmda.census.records.CensusRecords
+
+private[engine] object LarEngine2022 extends ValidationEngine[LoanApplicationRegister] {
+
+  override def syntacticalChecks(ctx: ValidationContext) = Vector(
+    S300,
+    S301.withContext(ctx)
+  )
+
+  override def validityChecks(ctx: ValidationContext) = Vector(
+    V600,
+    V608_1,
+    _2022.V608_2,
+    V609,
+    V610_1,
+    V610_2,
+    V611,
+    V612_1,
+    V612_2,
+    V613_1,
+    V613_2,
+    V613_3,
+    V613_4,
+    V614_1,
+    V614_2,
+    V614_3,
+    V614_4,
+    V615_1,
+    V615_2,
+    V615_3,
+    V616,
+    V617,
+    V618,
+    V619_1,
+    _2022.V619_2,
+    V619_3,
+    V620,
+    V621,
+    V622,
+    V623,
+    V624,
+    V625_1,
+    V625_2.withIndexedTracts(CensusRecords.indexedTract2022),
+    V626.withIndexedCounties(CensusRecords.indexedCounty2022),
+    V628_1,
```

Comment:
V627 and V716 are removed from the quarterly but we need to add them back for annual.